### PR TITLE
Enhance README and add specific logo

### PR DESCRIPTION
**Pull request by da-kami**
How it would look: https://github.com/comit-network/comit-js-sdk/blob/update-readme/README.md

Extended the README significantly to explain the purpose of the SDK and what one can do with it in more detail. 

Note that this README will also serve as the entry point to the SDK-documentation. (Some minor changes might be needed once we actually have the generated documentation.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,27 @@
-[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/comit-network/community)
+<a href="https://comit.network">
```
Comment (da-kami) [inline README.md]: Note: this link is to be updated to point to the SDK documentation section once it exists.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
+<img src="logo.svg" height="120px">
+</a>
 
-# COMIT Javascript SDK
+---
 
-A software development kit for developing Javascript applications on top of the COMIT protocol reference implementation [comit-rs](https://github.com/comit-network/comit-rs).
+[COMIT](https://comit.network) is an open protocol facilitating cross-blockchain applications.
+With [COMIT](https://comit.network) you can for examples exchange Bitcoin for any Erc20 token directly with another person.
```
Comment (bonomat) [inline README.md]: English grammar + small improvement proposal (feel free to edit): 
```suggestion
For example, with [COMIT](https://comit.network) you can exchange Bitcoin for Ether or any ERC20 token directly with another person.
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
 
-Currently [comit-rs](https://github.com/comit-network/comit-rs) is shipped as a binary called [cnd](https://github.com/comit-network/comit-rs/releases) (COMIT Network Daemon).
+This repository contains the Javascript software development kit (SDK) that wraps the communication with the [comit-network daemon](https://github.com/comit-network/comit-rs) for Javascript developers.
```
Comment (bonomat) [inline README.md]: Duplicated information - proposed change: 
```suggestion
This repository contains the Javascript software development kit (SDK) that wraps the communication with the [comit-network daemon](https://github.com/comit-network/comit-rs).
```

File: README.md [hunk 4]
```diff
@@ -1,13 +1,27 @@
...
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/comit-network/community)
+
+# COMIT Javascript SDK
+
+The COMIT Javascript SDK enables the integration of the [comit-network daemon](https://github.com/comit-network/comit-rs) into Javascript applications.
+You'll find the [latest comit-sdk version on npmjs](https://www.npmjs.com/package/comit-sdk).
+
+Below you find a short introduction into the SDK. For more detailed information please refer to the [API-documentation](TODO-add-link) in our documentation.
```
Comment (bonomat) [inline README.md]: Don't forget to track this TODO somewhere else. It's easy to lose track of this.

File: README.md [hunk 5]
```diff
@@ -16,16 +30,32 @@ Create a project using [create-comit-app](https://github.com/comit-network/creat
...
-* `sendSwaps`: when the taker initiates a swap with the maker (after successful negotiation)
-* `getNewSwaps`: when the maker has received a swap from the taker
-* `getOngoingSwaps`: for both maker and taker during the execution of the swap (fund, redeem)
-* `getDoneSwaps`: for both maker and taker to retrieve swaps that are finished
+### Trade Negotiation vs. Execution
 
-Additionally the SDK exports more fine-grained functionality to communicate with cnd, take a look at the classes in `cnd.ts` for details. 
+The SDK integrates a simple negotiation protocol that serves as a starting point for makers and takers to create and take orders.
+
+It is recommended to take a look at the [`TakerNegotiator`](https://github.com/comit-network/comit-js-sdk/blob/master/src/negotiation/taker_negotiator.ts) and [`MakerNegotiator`](https://github.com/comit-network/comit-js-sdk/blob/master/src/negotiation/maker_negotiator.ts) classes to get started.
```
Comment (bonomat) [inline README.md]: I would remove this sentence (or describe how to use it) as I would not know what to look at in these two files _to get started_.
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -34,3 +64,8 @@ The SDK includes two Javascript wallets:
 * Bitcoin wallet based on [bcoin](https://github.com/bcoin-org/bcoin)
 * Ethereum wallet based on [ethers.js](https://github.com/ethers-io/ethers.js/)
 
+These wallets are used to actually send the transactions necessary for the atomic swap to the respective ledger.
+COMIT is designed in a non-custodial way, thus `cnd` never holds your private key at any time.
+The wallets integrated in the SDK show how the transaction information received from `cnd` can be sent to the respective ledger.
+Note that you can, of course, use different wallets then the above default wallets in your application.
```
Comment (tcharding) [inline README.md]:
```suggestion
Note that you can, of course, use different wallets than the above default wallets in your application.
```

File: README.md (hunk 2 above)
Comment (tcharding) [reply README.md]: FWIW, I like @bonomat's edit.

File: README.md (hunk 4 above)
Comment (da-kami) [inline README.md]:
```suggestion
Below you find a short introduction into the SDK. For more detailed information please refer to the API-documentation in our documentation.
```

File: README.md (hunk 5 above)
Comment (da-kami) [inline README.md]:
```suggestion
The [`MakerNegotiator`](https://github.com/comit-network/comit-js-sdk/blob/master/src/negotiation/maker_negotiator.ts) class gives the maker the possibility to create and publish simple `JSON` orders.
The [`TakerNegotiator`](https://github.com/comit-network/comit-js-sdk/blob/master/src/negotiation/taker_negotiator.ts) class enables the taker to connect to the maker and take a published order. 
```

Comment (da-kami) [reply README.md]: @bonomat how about this?

File: README.md [hunk 7]
```diff
@@ -1,13 +1,27 @@
...
 
-A software development kit for developing Javascript applications on top of the COMIT protocol reference implementation [comit-rs](https://github.com/comit-network/comit-rs).
+[COMIT](https://comit.network) is an open protocol facilitating cross-blockchain applications.
+For example, with [COMIT](https://comit.network) you can exchange Bitcoin for Ether or any ERC20 token directly with another person.
 
-Currently [comit-rs](https://github.com/comit-network/comit-rs) is shipped as a binary called [cnd](https://github.com/comit-network/comit-rs/releases) (COMIT Network Daemon).
+This repository contains the Javascript software development kit (SDK) that wraps the communication with the [comit-network daemon](https://github.com/comit-network/comit-rs).
 
+If you wish to do an atomic swap on your machine or to integrate COMIT into an application (e.g. a DEX) please take a look at the [Getting Started section](https://comit.network/docs/getting-started/create-comit-app/) of the COMIT documentation.
 If you have any question please [reach out to the team in our Gitter chat](https://gitter.im/comit-network/community)!
```
Comment (da-kami) [inline README.md]:
```suggestion
If you have any questions, feel free to [reach out to the team in our Gitter chat](https://gitter.im/comit-network/community)!
```

File: README.md (hunk 5 above)
Comment (da-kami) [reply README.md]: Note: After discussion with @D4nte: 

The README is planned to serve as starting point for the documentation of the SDK, hence I would keep the information about `MakerNegotiator`, `TakerNegotiator` and `ComitClient` in there for now. If we decide to move the starting point of the SDK documentation to be separate from the README we can shorten the README - for now I would keep this information in here as an overview of the SDK (the information is in one place only at the moment, in the README).

Comment (da-kami) [reply README.md]: please see my comments below :)

File: README.md (hunk 4 above)
Comment (yosriady) [reply README.md]: Might be better to remove it until we have the final links.

File: README.md (hunk 5 above)
Comment (da-kami) [reply README.md]: since everyone else approved I'll just merge this PR for now. Follow-up PR if changes are needed.

**Pull request merged**